qiita_pet/handlers/study_handlers/edit_handlers.py

```diff
@@ -78,11 +78,13 @@ def __init__(self, study=None, **kwargs):
         if study:
             study_info = study.info
 
-            self.study_title.data = study.title
-            self.study_alias.data = study_info['study_alias']
+            self.study_title.data = study.title.decode('utf-8')
+            self.study_alias.data = study_info['study_alias'].decode('utf-8')
             self.pubmed_id.data = ",".join(study.pmids)
```

Comment:
This is still a freetext field, so it does have the possibility of having non UTF-8 characters. Better to add the decode.